app/web/features/search/SearchPage.tsx

```diff
@@ -104,12 +107,8 @@ export default function SearchPage({
         completeProfileFilter,
       ],
       ({ pageParam }) => {
-        const lastActiveComparation = parseInt(
-          lastActiveFilter as unknown as string
-        );
-        const hostingStatusFilterComparation = parseInt(
-          hostingStatusFilter as unknown as string
-        );
+        const lastActiveComparation = lastActiveFilter;
```

Comment:
same here, I remember I did this because when I was debugging the backend didn't receive the same values... is this checked?

Reply:
ok I checked and backend receive the same values... all in order!

Reply:
exactly, this creates a bug, there is a reason why I did this way, with your change if you use the lastActive filter (for example want to search lastActive "last day" and then want to unset this filter creates a bug)

![image](https://github.com/user-attachments/assets/9914e112-62bb-4cbb-841c-cda81441119b)

Reply:
that's why I created another issue (https://github.com/Couchers-org/couchers/issues/4565) and wanted to fix it there plus implement pagination

Reply:
so please restore :)

Reply:
Gotcha sorry about that will leave it to you then, I put it back with a ts-ignore and to-do comment.